O/openPMD_api/build_tarballs.jl

```diff
@@ -1,47 +1,93 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
+using Base.BinaryPlatforms
+const YGGDRASIL_DIR = "../.."
+include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "openPMD_api"
-version = v"0.14.3"
+version = v"0.15.0"
 
-julia_versions = [v"1.6.0", v"1.7.0", v"1.8.0"]
+# `v"1.6.3"` fails to build
+julia_versions = [v"1.7.0", v"1.8.0", v"1.9.0"]
```

Comment:
Does this mean this package should require Julia v1.7?  We should probably add https://github.com/JuliaPackaging/Yggdrasil/blob/f8adbcba50af81298d87fb945a65bb5383956a17/L/libsingular_julia/build_tarballs.jl#L16 to the fancy toys

Reply:
This comes from `libcxxwrap_julia_jll`. Ideally we would figure out why things fail for Julia 1.6, but that's not a high priority for me.

Yes, `julia_compat` should probably be set to `1.7` in this case. Or the more fancy thing you're suggesting; in practice, both should be the same (until there's a 1.10?).